src/lib/journey/callbacks/select-idp/select-idp.svelte

```diff
@@ -53,20 +57,39 @@
       value: option.provider,
       text: option.uiConfig.buttonDisplayName,
     }));
+
+    prefersDarkTheme = window.matchMedia('(prefers-color-scheme: dark)').matches;
   }
 </script>
 
-<Grid num={idps.length}>
-  {#each idps as idp}
-    <Button type="button" width="auto" onClick={() => setBtnValue(idp.value)}
-      ><AppleIcon
-        classes="tw_inline-block tw_fill-current tw_text-secondary-dark dark:tw_text-secondary-light"
-      />
-      <T key="signInWith" />
+{#each idps as idp}
+  <Grid num={1}>
+    <Button type="button" width="auto" onClick={() => setBtnValue(idp.value)}>
+      {#if idp.text.toUpperCase().includes('APPLE')}
+        {#if prefersDarkTheme}
+          <AppleIconLight
```

Comment:
If we lean on Tailwind and CSS only, it will allow us to use just use one Apple icon component and reduce the amount of Svelte code we have to bundle with the widget.

Reply:
Done.